src/CalculatorGeneralItem/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import Button from '../Button';
+import Spacer from '../Spacer';
+import Typography from '../Typography';
+import Wrapper from '../Wrapper';
+import styles from './styles.css';
+
+function CalculatorGeneralItem(props) {
+  return (
+    props.expanded
+    ? <div>
+      <Wrapper className={classNames(styles.wrapperExpanded, {[props.className]: props.className})}
+        size={'standard'}
+        >
+        <Typography
+          className={props.inactive ? styles.typoContentInactive : styles.typoQuestion}
+          type={'bodyTextNormal'}
+          >
+          {props.label}
+        </Typography>
+        <Button
+          className={styles.saveButton}
+          onClick={props.onSaveButtonClicked}
+          >
+          LAGRE
```

Comment:
You should send in text as props. This way we can use the language-files to translate it :-)